### PR TITLE
Loadin skeleton and home post cleanup

### PR DESCRIPTION
**Pull request by Gaelvego**
# Description

Included views for no posts, no history, no drafts, no internet connection. Included loading skeletons and animations for images and posts. Home posts correctly display information such as profile picture, username, location, etc...
Implemented svgs for tab icons and for missing image and missing post.

## Issues
#36 

## Screenshots
image
image

## Test
Turn off internet, in PostCard set error to true to test missing post functionality.

## Possible Downsides
Svgs are implemented are tsx component that return the corresponding svg tag, not svg files. Svgs seem to show up with thin lines even though they are copied directly from Figma. Even after images are fetched there is still a small delay until they properly appear on the screen.

## Additional Documentations
Added netInfo dependency for checking connection

### CODE REVIEW

File: src/components/home/HomePost.tsx [hunk 1]
```diff
@@ -29,56 +31,110 @@ interface HomePostProps {
...
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const data = await getOneUser(post.postedBy);
+        setPostedByData(data);
+        if (data.profile){
+          const url = await getDownloadURL(ref(storage, "profilePictures/" + data.profile));
+          setPostedByPic(url);
+        }
```
Comment (johnny-t06) [inline src/components/home/HomePost.tsx:79]: Is there a case where this might error out, where url is undefined?

**Review by johnny-t06 — COMMENTED** [review]
GJ team, it looks pretty good! I just have some small comments but the PR should be good after our requests

File: src/components/home/HomePost.tsx [hunk 2]
```diff
@@ -29,56 +31,110 @@ interface HomePostProps {
   style?: object;
 }
 
+const placeholder = require(" ../../../assets/icons/freebites/placeholder.png");
```
Comment (LordofMankid) [inline src/components/home/HomePost.tsx:34]: small nitpick but just move this up with the other imports - 'require' syntax is basically the same as import so we just want it all in one place

File: src/components/home/svg/ActiveHomeIcon.tsx
Comment (LordofMankid) [inline src/components/home/svg/ActiveHomeIcon.tsx:1]: can you move these icons into assets/icons/home? just so icons are all in one place

**Review by LordofMankid — APPROVED** [review]
lgtm after moving the svgs

**Pull request merged**